### DISCUSSION TITLE
Full window height

### DISCUSSION THREAD

**Discussion by Lilja** [wezterm/wezterm#2785] [Q&A] [answered]
I tried searching for this in both issues and discussions, but I couldn't find anything.

Is there a way to get full window height(and I suppose width too)? I fee like there's a bit of asymetrical "padding" for lack of a better word between the wezterm theme and my vim theme. See screenshot:
![theme](https://i.imgur.com/lRAf33h.png)

**Comment by wez** [comment, marked as answer]
Take a look at https://wezfurlong.org/wezterm/config/lua/config/window_padding.html

**Reply by Lilja** [reply]
Thanks, you rock!

**Reply by Lilja** [reply]
Looks a bit off though, i put some extra top padding.
![image](https://user-images.githubusercontent.com/6134511/203499995-eab9f7b0-d124-4c52-9b30-5e7cb93e280e.png)

Padding 0 on all axis:
![image](https://user-images.githubusercontent.com/6134511/203500192-514ca850-a709-4c7b-9761-4baff09db6fb.png)
